packages/core/src/stream-loading/__tests__/tip-fetcher.test.ts

```diff
@@ -20,6 +20,7 @@ describe('TipFetcher test', () => {
     ipfsClient = await createIPFS()
 
     dispatcher = await createDispatcher(ipfsClient, TOPIC)
+    const logger = new LoggerProvider().getDiagnosticsLogger()
```

Comment:
this seems to be a no-op?  No need to define this here given its never used and then immediately goes out of scope